cli_tools/gce_vm_image_import/importer/processor.go

```diff
@@ -26,6 +26,7 @@ import (
 type processor interface {
 	process(ctx context.Context) error
 	traceLogs() []string
+	cancel(reason string)
```

Comment:
have you envisioned other cancellation reasons?

Reply:
Yes. For example, customer cancelled. We currently rely on Daisy for these cancellations. Certain steps already don't use Daisy and we might need to implement that feature in the importer itself.

Reply:
sgtm